modules/mongodb/manifests/s3backup/backup.pp

```diff
@@ -66,6 +66,7 @@
   $user = 'govuk-backup'
 ){
 
+  require mongodb::s3backup::package
```

Comment:
This require should be attached to the `/usr/local/bin/mongodb-backup-s3` resource to avoid it being 'free floating.' You can also remove the explicit require on the user in that resource.